resources/services/elasticache/gen.hcl

```diff
@@ -0,0 +1,126 @@
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreCommonErrors"
+  }
+
+  deleteFilter "AccountRegionFilter" {
+    path = "github.com/cloudquery/cq-provider-aws/client.DeleteAccountRegionFilter"
+  }
+
+  multiplex "AwsAccountRegion" {
+    path   = "github.com/cloudquery/cq-provider-aws/client.ServiceAccountRegionMultiplexer"
+    params = ["elasticache"]
+  }
+
+  options {
+    primary_keys = ["arn"]
+  }
+
+  userDefinedColumn "account_id" {
+    description = "The AWS Account ID of the resource."
+    type        = "string"
+    resolver "resolveAWSAccount" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSAccount"
+    }
+  }
+
+  userDefinedColumn "region" {
+    type        = "string"
+    description = "The AWS Region of the resource."
+    resolver "resolveAWSRegion" {
+      path = "github.com/cloudquery/cq-provider-aws/client.ResolveAWSRegion"
+    }
+  }
+
+  column "auto_minor_version_upgrade" {
+    description = "Auto minor version upgrade"
+  }
+
+  column "cache_cluster_create_time" {
+    rename = "create_time"
+  }
+
+  column "cache_cluster_id" {
+    rename = "id"
+  }
+
+  column "cache_cluster_status" {
+    rename = "status"
+  }
+
+  column "notification_configuration_topic_arn" {
+    description = "The arn of a notification topic used for publishing ElastiCache events to subscribers using Amazon Simple Notification Service (SNS)"
+  }
+
+  column "notification_configuration_topic_status" {
+    description = "The current state of a notification topic used for publishing ElastiCache events to subscribers using Amazon Simple Notification Service (SNS)"
+  }
+
+  column "pending_modified_values_auth_token_status" {
+    rename = "pending_auth_token_status"
+    description = "Auth token status that is applied to the cluster in the future or is currently being applied"
+  }
+
+  column "pending_modified_values_cache_node_ids_to_remove" {
+    rename = "pending_cache_node_ids_to_remove"
+  }
+
+  column "pending_modified_values_cache_node_type" {
+    rename = "pending_cache_node_type"
+  }
+
+  column "pending_modified_values_engine_version" {
+    rename = "pending_engine_version"
+    description = "Cache engine version that is being applied to the cluster (or will be applied)"
+  }
+
+  column "pending_modified_values_num_cache_nodes" {
+    rename = "pending_num_cache_nodes"
+  }
+  
+  relation "aws" "elasticache" "cache_nodes" {
+    column "cache_node_create_time" {
+      rename = "create_time"
+    }
+
+    column "cache_node_id" {
+      rename = "id"
+    }
+
+    column "cache_node_status" {
+      rename = "status"
+    }
+  }
+
+  relation "aws" "elasticache" "cache_security_groups" {
+    column "cache_security_group_name" {
+      rename = "name"
+    }
+  }
+
+  relation "aws" "elasticache" "aws_elasticache_cluster_log_delivery_configurations" {
+    column "destination_details_cloud_watch_logs_details_log_group" {
+      rename = "cloudwatch_destination_log_group"
```

Comment:
It seems like you already addressed my comments from above here, but maybe the column name didn't match what cq-gen expected? I've been meaning to have cq-gen raise an error or warning when that happens, but it doesn't right now.

Reply:
wrong "relation name" was the bug - changed to 'log_delivery_configurations' and it worked